torch_xla/csrc/torch_util.h

```diff
@@ -25,6 +26,10 @@ struct SymIntElements {
   std::vector<torch::lazy::NodePtr> GetNodes() { return size_nodes_; }
   std::vector<int64_t> GetUpperBounds() { return upper_bounds_; }
   std::vector<bool> GetDynamicDims() { return dynamic_dims_; }
+  void SetUpperBound(int64_t index, int64_t upper_bound) {
+    XLA_CHECK_GT(upper_bounds_.size(), index);
+    return upper_bounds_[index] = upper_bound;
```

Comment:
this return syntax is a bit weird. I thought it was trying to return `upper_bound`. Can we just drop the `return` or put return in a separate line?

Reply:
lol - the return is redundant. oversight. let me remove it.